R/semanticPage.R

```diff
@@ -121,6 +121,28 @@ check_semantic_theme <- function(theme_css, full_url = TRUE) {
 #' \code{SUPPORTED_THEMES} or at http://semantic-ui-forest.com/themes.
 #' @param supress_bootstrap boolean flag that supresses bootstrap when turned on
 #' @param margin character with body margin size
+#' @examples
+#' ## Only run examples in interactive R sessions
+#' if (interactive()) {
+#' library(shiny)
+#' library(shiny.semantic)
+#'
+#' ui <- semanticPage(
+#'   title = "Hello Shiny Semantic!",
+#'   label("Number of observations:",
+#'         slider_input("obs", value = 500, min = 0, max = 1000)),
```

Comment:
`slider_input` shouldn't be wrapped by label tag:
```r
  label("Number of observations:"),
  slider_input("obs", value = 500, min = 0, max = 1000),
```

Reply:
fixed